tests/tests_selinux_modules.yml

```diff
@@ -0,0 +1,60 @@
+---
+- hosts: all
+  become: true
+  become_method: sudo
+  become_user: root
+  vars:
+    selinux_modules:
+      - { path: "linux-system-roles-selinux-test-a.cil" }
+      - { name: "linux-system-roles-selinux-test-a", state: "disabled" }
+      - { name: "linux-system-roles-selinux-test-a", state: "enabled" }
+      - { path: "linux-system-roles-selinux-test-b.cil", priority: "500" }
+      - { name: "linux-system-roles-selinux-test-b", priority: "500", state: "disabled" }
+      - { path: "linux-system-roles-selinux-test-c.cil", priority: "600" }
+      - { name: "linux-system-roles-selinux-test-c", priority: "600", state: "absent" }
+      - { name: "linux-system-roles-selinux-test-d", state: "disabled" }
+
+  tasks:
+    - name: execute the role and catch errors
+      block:
+        - include_role:
+            name: linux-system-roles.selinux
+
+    - name: Get status of linux-system-roles-selinux-test-a module
+      set_fact:
+        test_module: "{{ lookup(
+          'selinux_modules',
+          'linux-system-roles-selinux-test-a',
+          priority='400') }}"
+
+    - name: Check if linux-system-roles-selinux-test-a is installed and enabled
+      assert:
+        that: "{{ test_module.state == 'enabled' }}"
```

Comment:
you don't need `"{{ }}"` with `assert.that` or `when` - you should use 
`that: test_module.state == "enabled"`

Reply:
https://github.com/linux-system-roles/selinux/pull/56/commits/d3b629e070c6246cb65debf99951130b62b8f42b